content/blog/blog-post-08.md

```diff
@@ -1,27 +1,52 @@
+For Kokkos the US Department of Energy's (DOE) Exascale Computing Project changed that.
+By providing a framework that included all DOE's national laboratories it was possible to grow the Kokkos team beyond a single institution and establish a multi-laboratory collaborative effort to develop a robust performance portability solution.
 
+HPSF intents to be the common ground for establishing these collaborations on a global scale. 
+ 
+# HPSF Structure
 
-# Kokkos Is Open for Global Business
+HPSF will be controlled by a Governing Board and a Technical Advisory Council.
+The Governing Board will consist of a mix of representatives from HPSF member organizations and some representation from technical projects such as Kokkos.
+It decides how to allocate the budget of the foundation, and also controls marketing and HPSF events.
+Furthermore, the Governing Board decides on general HPSF rules.
 
-Key benefits of belonging to the Linux Foundation High-Performance Software Foundation include open governance models, resources for improving software quality and building community, as well as the association with the larger Linux computing ecosystem.
-Kokkos welcomes all manner of high-quality contribution -- from fixes and features, to build system upgrades, and even pioneering capabilities in new domains.
-If you feel inspired to contribute, please reach out on the [Kokkos Slack Channel](https://kokkos.org/kokkos-core-wiki/faq.html) to discuss.    
+The Technical advisory committee organizes technical working groups and develops criteria for the project lifecycle in HPSF. It consists mainly of representatives of the technical projects.
+Technical governance of projects associated with HPSF is still up to each individual project.
+The Advisory Council is also the primary body to organize day-to-day collaboration with other Linux Foundation efforts.
 
+{{< image src="img/blog/2024/hpsf-structure.png" style="float: center; height=10">}}
 
-# Impacts, Benefits for Kokkos Users, Developers
 
-Kokkos developers and users will likely experience, few, if any, change in their day-to-day activities, but over the longer term, will realize the benefits of improved software, open governance, resources for community building, and world-wide collaboration.
+# Open for Global Business
+
+Key benefits of belonging to the High-Performance Software Foundation include open governance models, resources for improving software quality and building community, as well as collaborations with the larger Linux computing ecosystem.
+Leveraging well established open governance models from the Linux Foundation, HPSF provides a framework to organize multi-institutional collaborations.
+An institution that wants to leverage Kokkos and also become contributors can now rest assured that the project is not just subject to the whims of some project leads paid for by DOE.
```

Comment:
Suggestion:

Instead of "... subject to the whims ...",


Try:


"Operationally, Linux Foundation open governance means the projects are truly open and available to anyone at any institution, unencumbered by original ownership pedigrees."

Reply:
I agree, we probably want to be more diplomatic :P